int_experian/int_experian_sgjc/cartridge/static/default/js/edqUtils.js

```diff
@@ -418,39 +431,45 @@ function edqSetGlobalIntuitiveConfiguration() {
 	];
 }
 /**
-* Pro Web - Address (Verification Engine)
-* Sets the configuration to use Pro Web - Address (Verification Engine)
-*/
+ * Pro Web Address Verification callback
+ */
 function edqValidateAddressCallBack() {
 	var edqProWebResponse = document.querySelector("#form-submit");
 	if (edqProWebResponse.hasAttribute("edq-metadata")) {
 		var edqProWebMetaDataJSON = JSON.parse(edqProWebResponse.getAttribute("edq-metadata"));
 		document.querySelector("#form-submit").style.display = "none";
-		edqStateLineSelector.value = edqProWebMetaDataJSON["State code"];
+		edqStateLineSelector.value = edqProWebMetaDataJSON["address.province"];
 	}
 	if (edqProWebCallbackValidation) {
 		if (edqProWebExecuteTransitionCallBack(edqProWebMetaDataJSON)) {
 			edqCheckoutPageWorkflows();
 		}
 	} else { edqCheckoutPageWorkflows(); }
 }
+/**
+ * Manage the button settings for the Button "form-submit", depending on the touchpoint.
+ */
 function edqCheckoutPageWorkflows() {
-	if (edqCurrentSubmitButtonSelectorIndex) {
-		buttonSelector[edqCurrentSubmitButtonSelectorIndex].style.display = "inline-block";
-		buttonSelector[edqCurrentSubmitButtonSelectorIndex].click();
+	if (edqCurrentSubmitButtonIndex) {
+		buttonSelector[edqCurrentSubmitButtonIndex].style.display = "inline-block";
+		buttonSelector[edqCurrentSubmitButtonIndex].click();
 	} else {
-		edqCurrentSubmitButtonSelector.style.display = "inline-block";
-		edqCurrentSubmitButtonSelector.click();
+		edqCurrentSubmitButton.style.display = "inline-block";
+		edqCurrentSubmitButton.click();
```

Comment:
Have you tested `click()` on Internet Explorer? I know some of these methods aren't 100% cross compatible with all of the browsers.

Reply:
Yes, it has compatibility with all browser.